dephell/config/builders.py

```diff
@@ -72,6 +72,7 @@ def build_other(parser):
     other_group = parser.add_argument_group('Other')
 
     other_group.add_argument('--owner', help='name of the owner.')
+    other_group.add_argument('--tag', help='create git tag.')
```

Comment:
`action='store_true'`

And you forgot to add it into config's validation scheme

Reply:
Ah sure I'll do that.